src/pages/docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -85,19 +87,19 @@ Once the variables for target discovery have been configured the **Amazon Web Se
 
 Octopus looks for tags applied to cloud resources to discover and create deployment targets for you.
 
-Tags are in the format `octopus-{scope}` and support the following for discovery. Note that only a single value is supported in tags at the moment.
+Tags in cloud resource templates are in the format `octopus-{scope}` and support the following for discovery. Note that only a single value is supported in tags at the moment.
 
 | Tag                   | Required | Description                                                                                                                                                                                  | Example                             |
 | --------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
 | `octopus-environment` | Yes      | The name of the [environment](/docs/infrastructure/environments) the target can be used in during deployments. Only deployments matching the environment will discover the target.  | `octopus-environment = Development` |
-| `octopus-role`        | Yes      | The [role](/docs/infrastructure/deployment-targets/#target-roles) that should be applied to the target. Only deployments with a step that matches the role will discover the target. | `octopus-role = PetShotFrontEnd`    |
+| `octopus-role`        | Yes      | The [target tag](/docs/infrastructure/deployment-targets/#target-roles) that should be applied to the target. Only deployments with a step that matches the target tag will discover the target. | `octopus-role = PetShotFrontEnd`    |
 | `octopus-space`       | No       | The name of the [space](/docs/administration/spaces) the target can be used in. If present only deployments within the matching space can discover the target.                      | `octopus-space = PetShopTeam`       |
 | `octopus-project`     | No       | The name of the [project](/docs/projects) the target will be discovered by. If present only deployments for the matching project can discover the target.                           | `octopus-project = PetShop`         |
 | `octopus-tenant`      | No       | The name of the [tenant](/docs/projects) the target can be discovered for. If present only deployments for the matching tenant will discover the target.                            | `octopus-tenant = MyClient`         |
 
 ## Add step to deployment process
 
-Octopus will discover targets if one of the following steps are in your deployment process. Each step will discover targets that match it's target role.
+Octopus will discover targets if one of the following steps are in your deployment process. Each step will discover targets that match it's target tags.
```

Comment:
```suggestion
Octopus will discover targets if one of the following steps are in your deployment process. Each step will discover targets that match its target tags.
```

Reply:
Oops, I missed making this change before merging the PR 

I will include this fix in the next batch of changes